research_cloud/environment.yml

```
@@ -0,0 +1,21 @@
+---
+apt:
+  repo_keys: []
+  repos:
+  - 'deb http://archive.ubuntu.com/ubuntu focal main multiverse universe restricted'
+  - 'deb http://archive.ubuntu.com/ubuntu focal-security main multiverse universe restricted'
+  packages:
+  - ncdu
+
+miniconda:
+  name: tempo-embeddings
+  channels:
+  - conda-forge
+  dependencies:
+  - python=3.9      # change version if needed
```

Could we include more recent Python versions?

```suggestion
  - python>=3.9      # change version if needed
```